src/Model/test.js

```diff
@@ -127,6 +127,7 @@ describe('watermelondb/Model', () => {
   it('can update a record', async () => {
     const database = makeDatabase()
     database.adapter.batch = jest.fn()
```

Comment:
why is this line even here? Is it necessary? is `database` a mock? or is it a real `Database` object?